src/AlgoliaSearch.js

```diff
@@ -653,6 +653,52 @@ AlgoliaSearch.prototype.searchUserIDs = function(data, callback) {
   });
 };
 
+/**
+ * Set strategy for personalization
+ *
+ * @param {Object} data
+ * @param {Object} data.eventsScoring Scoring to apply to events
+ * @param {Object} data.facetsScoring Scoring to apply on facets
```

Comment:
I would have liked further documenting the content of theses objects but I don't see a pattern for doing so in the rest of the codebase.
Any inputs @Haroenv ?

Reply:
is eventsScoring an object? please put the shape in the documentation, and not just a repeat of the name :)

Reply:
The example a few lines after demonstrates the shape of the object.
Could something like this be both understandable and picked up during docs parsing?
```
* @param {Object<eventName, {score: number, type: "conversion"|"click"|"view"}>} data.eventsScoring Scoring to apply to events
 ```

Reply:
you need to go deeper like you did with data.eventsScoring, e.g. data.eventsScoring.eventName